File: code/coding_pratice_Interview/count_vowels/count_vowels.py
def count_vowels(value):
    vowels = {'a', 'e', 'i', 'o', 'u'}
    count = 0;
    for letter in value:
        if(letter in vowels):
            count += 1
    print("The Total numbers of Vowels are: ", count)


count_vowels("education")

File: code/coding_pratice_Interview/count_vowels/count_vowels.js
const count_vowels = (value) =>{
    count = 0;
    for(let i = 0; i < value.length; i++){
        if(value.charAt(i) == "a" || value.charAt(i) == "e" || value.charAt(i) == "i" || value.charAt(i) == "o" || value.charAt(i) == "u"){
            count += 1
        }
    }   
    console.log(count);
}
    
// count_vowels("education")